components/ActionSheet/tests/snapshot/MDCActionSheetControllerInsetsAPIsSnapshotTests.m

```diff
@@ -323,4 +323,161 @@ - (void)testContentEdgeInsetsShiftActionRowToRightInRTL {
+  [controller addAction:self.action3];
+  [controller addAction:self.action4];
+  [controller addAction:self.action5];
+
+  // When
+  controller.view.bounds = CGRectMake(0, 0, 320, 640);
+  controller.imageEdgeInsets = UIEdgeInsetsZero;
+
+  // Then
+  [self generateSnapshotAndVerifyForView:controller.view];
+}
+
+- (void)testImageEdgeInsetsTopInsetMovesImageViewOutsideCell {
+  // Given
+  [self changeToLongLatinTitles];
+
+  MDCActionSheetController *controller =
+      [MDCActionSheetController actionSheetControllerWithTitle:kLongTitle4Latin
+                                                       message:kLongTitle2Latin];
+  [controller addAction:self.action1];
+  [controller addAction:self.action2];
+  [controller addAction:self.action3];
+  [controller addAction:self.action4];
+  [controller addAction:self.action5];
+
+  // When
+  controller.view.bounds = CGRectMake(0, 0, 320, 640);
+  controller.imageEdgeInsets = UIEdgeInsetsMake(12, 0, 0, 0);
+
+  // Then
+  [self generateSnapshotAndVerifyForView:controller.view];
+}
+
+- (void)testImageEdgeInsetsShiftIconToRightInLTR {
+  // Given
+  [self changeToLongLatinTitles];
+
+  MDCActionSheetController *controller =
+      [MDCActionSheetController actionSheetControllerWithTitle:kLongTitle4Latin
+                                                       message:kLongTitle2Latin];
+  [controller addAction:self.action1];
+  [controller addAction:self.action2];
+  [controller addAction:self.action3];
+  [controller addAction:self.action4];
+  [controller addAction:self.action5];
+
+  // When
+  controller.view.bounds = CGRectMake(0, 0, 320, 640);
+  controller.imageEdgeInsets =
+      UIEdgeInsetsMake(controller.imageEdgeInsets.top, -50, controller.imageEdgeInsets.bottom, 50);
+
+  // Then
+  [self generateSnapshotAndVerifyForView:controller.view];
+}
+
+- (void)testImageEdgeInsetsShiftIconToLeftInLTR {
+  // Given
+  [self changeToLongLatinTitles];
+
+  MDCActionSheetController *controller =
+      [MDCActionSheetController actionSheetControllerWithTitle:kLongTitle4Latin
+                                                       message:kLongTitle2Latin];
+  [controller addAction:self.action1];
+  [controller addAction:self.action2];
+  [controller addAction:self.action3];
+  [controller addAction:self.action4];
+  [controller addAction:self.action5];
+
+  // When
+  controller.view.bounds = CGRectMake(0, 0, 320, 640);
+  controller.imageEdgeInsets =
+      UIEdgeInsetsMake(controller.imageEdgeInsets.top, 20, controller.imageEdgeInsets.bottom, -20);
+
+  // Then
+  [self generateSnapshotAndVerifyForView:controller.view];
+}
+
+- (void)testImageEdgeInsetsShiftIconToLeftInRTL {
+  // Given
+  [self changeToLongArabicTitles];
+
+  MDCActionSheetController *controller =
+      [MDCActionSheetController actionSheetControllerWithTitle:kLongTitle4Arabic
+                                                       message:kLongTitle2Arabic];
+  [controller addAction:self.action1];
```

Comment:
Can we make a method `addFiveActions` and then call that every time instead of calling all of them?

Reply:
Sure! I will do that in a refactor PR after this one.

Reply:
Done in #9029, thanks!